backend/src/plugins/Automod/actions/pauseInvites.ts

```diff
@@ -0,0 +1,14 @@
+import * as t from "io-ts";
+import { automodAction } from "../helpers";
+
+export const PauseInvitesAction = automodAction({
+  configType: t.type({
+    paused: t.boolean,
+  }),
+
+  defaultConfig: {},
+
+  async apply({ pluginData, actionConfig }) {
+    await pluginData.guild.disableInvites(actionConfig.paused);
+  },
```

Comment:
I feel like with this action name, there is no longer an ambiguity towards what it will do, so this should be enough

```suggestion
  configType: t.boolean,

  defaultConfig: true,

  async apply({ pluginData, actionConfig }) {
    await pluginData.guild.disableInvites(actionConfig);
  },
```

Reply:
I had this originally, but it seems Zeppelin ignores falsy actions. It was able to pause invites when set to `true`, but the action wouldn't run when `false`

Reply:
Interesting
https://github.com/ZeppelinBot/Zeppelin/blob/5a4e50b19de8f19898826ef2e58c6865c03b90c5/backend/src/plugins/Automod/functions/runAutomod.ts#L133-L135